codegen/permission-generator/Readme.md

```diff
@@ -0,0 +1,89 @@
+# Permission Generator
```

Comment:
This is such a stupid nitpick but can you make sure the filename here is `README.md` instead of `Readme.md`?

Reply:
done ✅